go-controller/pkg/ovn/ovn.go

```diff
@@ -754,6 +752,33 @@ func (oc *Controller) removeServiceEndpoints(lb, vip string) {
 	}
 }
 
+func (oc *Controller) assignPodAddresses(nodeName string) (net.HardwareAddr, *net.IPNet, error) {
+	var (
+		podMac    net.HardwareAddr
+		podIP     *net.IP
+		podIPMask *net.IPMask
+		err       error
+	)
+
+	// First try to get the pod addresses quickly then fall back to polling every second.
+	podIP, podIPMask, err = oc.lsCache.AllocateNextIP(nodeName)
+	if err != nil {
+		return nil, nil, err
+	}
+	podMac, err = hwaddr.GenerateHardwareAddr4(*podIP, hwaddr.PrivateMACPrefix)
+	if err != nil {
+		defer func() {
```

Comment:
No need to defer here since this isn't a big function and there's only one error path after the allocation.

Reply:
Also Winship is going to yell about "podMac". If we're capitalizing the IP in podIP, then we should capitalize the MAC in podMac

Reply:
will change to `podMAC`.